internal/core/items/text.rs

```diff
@@ -325,10 +325,15 @@ impl Item for TextInput {
                     WindowInner::from_pub(window_adapter.window()).set_focus_item(self_rc);
                 }
             }
-            MouseEvent::Released { button: PointerEventButton::Left, .. } | MouseEvent::Exit => {
+            MouseEvent::Released { button: PointerEventButton::Left, .. } => {
+                self.as_ref().pressed.set(false)
+            }
+            MouseEvent::Exit => {
+                window_adapter.set_mouse_cursor(super::MouseCursor::Default);
                 self.as_ref().pressed.set(false)
             }
             MouseEvent::Moved { position } => {
+                window_adapter.set_mouse_cursor(super::MouseCursor::Text);
```

Comment:
I think this should restore instead of override. For example if a TextInput is a child of a TouchArea with a custom cursor.

Reply:
Sorry this was meant for the Exit branch :)

Reply:
There is no "restore" function.
Also, the TouchArea will also set the mouse cursor on every mouse move event, so it should be good.

Reply:
Ahh good point:)